docker/tests/curl/Dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG FROM_IMAGE
+FROM ${FROM_IMAGE} as prepare-curl
+
+WORKDIR /srv
+
+# Installs node, npm and newman
+ENV NODE_VERSION=12.6.0
+RUN apt-get install -y curl
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5442148)